README.md

```diff
@@ -54,7 +54,7 @@ To persist `libphonenumber\PhoneNumber` objects, add the `Misd\PhoneNumberBundle
     doctrine:
         dbal:
             types:
-                phone_number: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
+                phonenumber: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
```

Comment:
This does now not match the type name used below anymore.

Reply:
@adequat-bi - @xabbuh is right, I think you're confusing the field/property name with the name of the type. `phone_number` is the correct name of the type. Closing.